app/assets/javascripts/discourse/app/services/group-automatic-members-dialog.js

```diff
@@ -8,12 +8,12 @@ export default class GroupAutomaticMembersDialog extends Service {
   @service dialog;
 
   async showConfirm(group_id, email_domains) {
-    const domainCount = email_domains?.split("|")?.length ?? 0;
-
-    if (domainCount === 0) {
+    if (!email_domains) {
```

Comment:
Could be worth using `isEmpty` for this, but also this is fine https://api.emberjs.com/ember/release/functions/@ember%2Futils/isEmpty